Documentation/docs/source/building.rst

```diff
@@ -197,14 +197,27 @@ After proper configuration, SimpleITK can be built:
 Advanced Build Options
 ----------------------
 
-SimpleITK is aware of the enabled and disabled ITK Modules. It is
-possible to customize SimpleITK by disabling unneeded ITK modules or to
-extend SimpleITK by writing custom json file dependent an ITK Remote
-Modules being enabled.
-
-Additionally, each of the Examples directory, and each of the Wrapping
-subdirectories can be configured as independent CMake project dependent
-on an installed SimpleITK.
+SimpleITK is aware of the modularity of ITK and automatically enables
+and disables filters based on which modules are available from the ITK
+SimpleITK is compiled against. This makes is possible to customize
+SimpleITK to be a small library or to wrap additional ITK remote
+modules simply configuring ITK with the desired modules enabled.
+
+For example, the CoherenceEnhancingDiffusionImageFilter is an optional
+filter in SimpleITK as it's from the ITK remote module
+AnisotropicDiffusionLBR. This remote module is not enabled by default
+when building ITK and SimpleITK. To enabled it when using the
+Superbuild add `-DModule_AnisotropicDiffusionLBR:BOOL=ON` to the
+command line or in the CMake GUI press the "Add Entry" button to
+define the variable as above.
+
+SimpleITK has a very flexible and robust build system utilizing
+CMake. It enables packages to build SimpleITK in a variety of ways to
+suit their requirements and minimize recompilation of SimpleITK so
+that it can be wrapped for many different languages. Each of the
+language wrapping sub-directories i.e. "Wrapping/Python" can be
+configured and build as an independent project which is dependent on
+SimpleITK as an install packaged of is libraries and header filers.
```

Comment:
...configured and **built** as an independent project which is dependent on **the installed SimpleITK libraries and header files.**